packages/wonder-blocks-theming/src/types.ts

```diff
@@ -1,3 +1,6 @@
 import {StyleDeclaration} from "aphrodite";
 
 export type ThemedStylesFn<T> = (theme: T) => StyleDeclaration;
+
+export type SupportedThemes = "default" | "khanmigo";
+export type Themes<T> = Partial<Record<SupportedThemes, T>>;
```

Comment:
**question:** Is `T` here an object? I wonder if we need to be specific about that with `T extends Object`?

Reply:
nice catch! I'll add that, and yes, it expects an object.